src/components/views/messages/MVideoBody.tsx

```diff
@@ -57,7 +59,28 @@ export default class MVideoBody extends React.PureComponent<IBodyProps, IState>
         };
     }
 
-    thumbScale(fullWidth: number, fullHeight: number, thumbWidth = 480, thumbHeight = 360) {
+    private get suggestedDimensions(): { w: number, h: number } {
+        switch (SettingsStore.getValue("Images.size") as ImageSize) {
+            case ImageSize.Large:
+                return { w: 480, h: 360 };
```

Comment:
This is now handled by constants in the `ImageSize` file.
Since the design team requested, that videos and images are the same size in the timeline, those values get used for both.
Of course first checking the actual image size constraints and all the other parameters, like requests for a custom height.